### PR TITLE
Bump the template version

### PR DESCRIPTION
**Pull request by sima-zhu**
Kick off the pipeline of template to test the release tag default setting. All passed, but release tag 1.2.1-beta.2 has been used. Bump to a new one to 1.2.1-beta.3.
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=525027&view=results

### CODE REVIEW

**Comment by weshaggard** [comment]
doesn't look like you used the version update script https://github.com/Azure/azure-sdk-for-java/blob/master/CONTRIBUTING.md#what-does-the-process-look-like.

**Comment by weshaggard** [comment]
Did you run the update script? It looks like there is still one reference in the repo without the updated version.

File: sdk/template/azure-sdk-template/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # Release History
+## 1.2.1-beta.3 (2020-09-08)
```
Comment (weshaggard) [inline sdk/template/azure-sdk-template/CHANGELOG.md:2]: Should probably be 'unreleased'

Comment (sima-zhu) [reply sdk/template/azure-sdk-template/CHANGELOG.md:2]: This is how update-version.py did. I am not sure why.

Comment (sima-zhu) [reply sdk/template/azure-sdk-template/CHANGELOG.md]: I change it back manually. Will take a look why the script not return me the right thing

**Review by weshaggard — APPROVED** [review]

File: sdk/template/azure-sdk-template/CHANGELOG.md (hunk 1 above)
Comment (weshaggard) [reply sdk/template/azure-sdk-template/CHANGELOG.md]: I suspect there might be another options that is needed to make this unreleased when bumping the version.  Did you do the same command as in https://github.com/Azure/azure-sdk-for-java/blob/master/eng/pipelines/templates/stages/archetype-java-release.yml#L235?

Comment (sima-zhu) [reply sdk/template/azure-sdk-template/CHANGELOG.md]: I did the set-version step right, but when execute update version step. I did not put the parameter --sr
python eng/versioning/update_versions.py --update-type library --build-type ${{ parameters.SDKType }} --avi

Comment (sima-zhu) [reply sdk/template/azure-sdk-template/CHANGELOG.md]: Tried put the --sr back. It works as expected.

**Pull request merged**